src/main/java/edu/ksu/canvas/attendance/model/AttendanceSummaryModel.java

```diff
@@ -27,15 +28,17 @@
         private final int totalClassesMissed;
         private final int totalClassesExcused;
         private final int totalClassesTardy;
+        private final int totalClassesPresent;
 
 
         public Entry(long courseId, long sectionId, long studentId,
-                     String studentName, boolean dropped, int sumMinutesMadeup, int remainingMinutesMadeup,
+                     String SisUserId, String studentName, boolean dropped, int sumMinutesMadeup, int remainingMinutesMadeup,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this local variable name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00117)